sphinx/writers/latex.py

```diff
@@ -1833,6 +1840,8 @@ def warner(msg):
             hlcode = hlcode.rstrip()[:-14]  # strip \end{Verbatim}
             self.body.append('\n' + hlcode + '\\end{%sVerbatim}\n' %
                              (self.table and 'Original' or ''))
+            if ids:
+                self.body.append('\\let\\SphinxLiteralBlockLabel\empty\n')
```

Comment:
The definition of `\SphinxLiteralBlockLabel` is done before `Verbatim` environment is entered in the produced LaTeX file. It is reset after the `Verbatim` environment has ended. This resetting is needed because next literal block will check the status of `\SphinxLiteralBlockLabel` in case no title was explicitly set. Thus if we did not reset to `\empty` that macro a second target with the same label could be created by another literal block with no caption.